danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/danger/DangerDSL.kt

```diff
@@ -1,9 +1,13 @@
-package systems.danger.kotlin
+package systems.danger.kotlin.models.danger
 
 import kotlinx.serialization.*
+import systems.danger.kotlin.models.bitbucket.BitBucketServer
+import systems.danger.kotlin.models.git.Git
+import systems.danger.kotlin.models.github.GitHub
+import systems.danger.kotlin.models.gitlab.GitLab
 
 @Serializable
-data class DSL(
+internal data class DSL(
```

Comment:
the `DSL` object shouldn't be visible from the Danger File, `DangerDSL` yes

Reply:
I always forget that in kotlin default is public, because on Swift the default is internal 😅